### ISSUE TITLE
Feature Request: Blended GEV

### ISSUE THREAD

**Issue by jdossgollin** [JuliaStats/Distributions.jl#1364]
Is there any possibility to implement the Blended GEV distribution derived by Castro-Camilo and coauthors?

> Castro-Camilo, D., Huser, R., & Rue, H. (2021). Practical strategies for GEV-based regression models for extremes. ArXiv:2106.13110 [Stat]. Retrieved from http://arxiv.org/abs/2106.13110

A tutorial is also available at https://inla.r-inla-download.org/r-inla.org/doc/vignettes/bGEVtutorial1-2.pdf. Fitting the distribution can prove tricky, but that's sort of beyond the scope of `Distributions.jl`; the `cdf` and `pdf` are defined in the paper.

**Comment by azev77** [comment]
Also checkout @jojal5
https://github.com/jojal5/Extremes.jl
he has nice methods for fitting GEV

**Comment by jdossgollin** [comment]
Yes, I've seen it. This is a reparameterization of the GEV rather than an estimation method -- some are suggested in the paper, but that's beyond the scope of `Distributions.jl`

**Comment by azev77** [comment]
Interesting bc they claim `gevfit()` estimates the parameters via ML:
https://github.com/jojal5/Extremes.jl/blob/master/src/parameterestimation/maximumlikelihood/maximumlikelihood_gev.jl

**Comment by jdossgollin** [comment]
It does 🙂
I'm saying that the Blended GEV (see paper I linked originally) is a reparameterization rather than an estimation method.
